mivisionx/PKGBUILD

```diff
@@ -1,22 +1,26 @@
 # Maintainer: acxz <akashpatel2008 at yahoo dot com>
+# Contributer: JP-Ellis <josh@jpellis.me>
```

Comment:
Could you please add me as co-maintainer?

Reply:
Technically it should be the following: